### PR TITLE
Fixes for client

### PR DESCRIPTION
**Pull request by lukeperrin10**
pt Story: https://www.pivotaltracker.com/story/show/178244856

Frontend team has requested some changes that they want API to do

## Suggest changes 
- Changes articles to be shown by updated_at rather than creation 
- adds author to index action 
- seeds ratings for articles. 

## What I learned 
- How to sort and array of hashes

### CODE REVIEW

File: app/controllers/api/articles_controller.rb [hunk 1]
```diff
@@ -3,9 +3,9 @@ class Api::ArticlesController < ApplicationController
   before_action :role_authenticator, only: %i[create]
 
   def index
-    articles = Article.all.most_recent
+    articles = Article.all.sort_by { |article| article[:updated_at] }
     if current_user&.journalist?
-      articles_by_journalist = Article.where(user_id: current_user.id).most_recent
+      articles_by_journalist = Article.where(user_id: current_user.id).sort_by { |article| article[:updated_at] }.reverse
```
Comment (LucasKnudsen) [inline app/controllers/api/articles_controller.rb]: If you guys check how we made the `.most_recent`, you could do the same with this :D

**Review by DavDan1 — APPROVED** [review]

File: app/serializers/articles_index_serializer.rb [hunk 2]
```diff
@@ -16,4 +16,8 @@ def image
   def rating
     Rating.where(article_id: object.id).average(:rating).to_f
   end
+
+  def author
+    "#{object.user['first_name']} #{object.user['last_name']}" 
+  end
```
Comment (LucasKnudsen) [inline app/serializers/articles_index_serializer.rb]: Would be nice to have it in the same format as in the old show action, so we don't have to change it on the clients :))

**Comment by LucasKnudsen** [comment]
Also tests are failing cuz you changed the format of author :3

File: app/serializers/articles_index_serializer.rb (hunk 2 above)
Comment (lukeperrin10) [reply app/serializers/articles_index_serializer.rb]: Fixed.

File: app/controllers/api/articles_controller.rb (hunk 1 above)
Comment (lukeperrin10) [reply app/controllers/api/articles_controller.rb]: Also Fixed.

**Review by LucasKnudsen — APPROVED** [review]

**Pull request merged**